src/Orchard.Web/Modules/Orchard.Users/Models/RegistrationSettingsPart.cs

```diff
@@ -42,5 +45,56 @@ public class RegistrationSettingsPart : ContentPart {
             set { this.Store(x => x.EnableLostPassword, value); }
         }
 
+        public bool EnableCustomPasswordPolicy {
+            get { return this.Retrieve(x => x.EnableCustomPasswordPolicy); }
+            set { this.Store(x => x.EnableCustomPasswordPolicy, value); }
+        }
+
+        public bool EnableCustomPasswordLength {
+            get { return this.Retrieve(x => x.EnableCustomPasswordLength); }
+            set { this.Store(x => x.EnableCustomPasswordLength, value); }
+        }
+
+        [Range(1, int.MaxValue, ErrorMessage = "The minimum password length must be at least 1.")]
```

Comment:
Make it a textbox that is always visible/editable. Today it's driven by `EnableCustomPasswordLength` which is then removed.